addons/mail/static/tests/composer/html_composer_message_field.test.js

```diff
@@ -86,9 +87,9 @@ test("media dialog: upload", async function () {
         return attachment;
     });
 
-    onRpc("/web/dataset/call_kw/ir.attachment/generate_access_token", () => {
-        return ["129a52e1-6bf2-470a-830e-8e368b022e13"];
-    });
+    onRpc("/web/dataset/call_kw/ir.attachment/generate_access_token", () => [
+        "129a52e1-6bf2-470a-830e-8e368b022e13",
+    ]);
```

Comment:
Avoid linting on unrelated lines, it breaks blame history for no reason.

Run commits with --no-verify or disable commit hooks temporarily if you're using the web tooling setup